src/udev/udev-builtin-input_id.c

```diff
@@ -206,6 +206,7 @@ static bool test_pointers(struct udev_device *dev,
          * rudders/pedals are joystick-like, but buttonless; they have
          * other fancy axes */
         has_joystick_axes_or_buttons = test_bit(BTN_TRIGGER, bitmask_key) ||
+                                       test_bit(BTN_TRIGGER_HAPPY, bitmask_key) ||
```

Comment:
sounds like a test we should eventually turn into some constant array and for loop...